Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM php:8.0.19-fpm-alpine
+FROM php:8.2.12-fpm-alpine
```

Comment:
raw question, why not 8.3 as of today stable php release ?

Reply:
Because this PR has been here since July 2023, that wasn't available back then. I'd like to keep it scoped for now. It's easy to make another PR later on 👍

Reply:
oki got it :)
(coming from the other PR related to php v8 👍🏻 )